app/src/pages/SignUp.tsx

```diff
@@ -1,272 +1,285 @@
-import * as React from "react"
-import { useNavigate } from "react-router-dom"
-import { LoadingButton } from "@mui/lab"
-import Avatar from "@mui/material/Avatar"
-import CssBaseline from "@mui/material/CssBaseline"
-import Grid from "@mui/material/Grid"
-import Box from "@mui/material/Box"
-import LockOutlinedIcon from "@mui/icons-material/LockOutlined"
-import Typography from "@mui/material/Typography"
-import Container from "@mui/material/Container"
-import { createTheme, ThemeProvider } from "@mui/material/styles"
-import Stepper from "@mui/material/Stepper"
-import Step from "@mui/material/Step"
-import StepLabel from "@mui/material/StepLabel"
-import Button from "@mui/material/Button"
+import * as React from "react";
```

Comment:
Remind me please, why we use import * as React from "react", but not use import React from "react"?

Reply:
I suggested to import this way like:
import { useEffect, useState } from 'react'

but i haven't fix it in this task

Reply:
of cource, it not a problem. 
Okey, maybe we will come back to this question later